packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -15,10 +16,13 @@ export async function findProjectRootAsync(cwd?: string): Promise<string | null>
   return projectRootDir ?? null;
 }
 
-export async function getProjectIdAsync(projectDir: string): Promise<string> {
+export const getProjectIdAsync = once(_getProjectIdAsync);
```

Comment:
I think we should not use `once` here, it can cause many kinds of issues.

The most obvious problem is that the arguments of subsequent calls are completely ignored. Example:
```ts
await getProjectIdAsync('/home/ville/super-important-project');

// Later...
const id = await getProjectIdAsync('/home/ville/hobby-project');
await permanentlyDeleteProjectAsync(id); // Deletes super-important-project 😞 
```

Additionally using `once` to cache the result creates a hidden cache inside the function that can be hard to invalidate and can cause surprising results in tests, for example. We've also had some discussions about using the cache of the GraphQL client, which is more widely applicable and allows setting up a normalized cache that will ensure the cached value is updated if a following query returns a new value.

I would suggest just making sure each command only calls this function once and stores the result in a variable passed to other functions.